### DISCUSSION TITLE
How to apply for student pack when homeschooling?

### DISCUSSION THREAD

**Discussion by ErikApption** [community/community#43748] [GitHub Education]
Hi,

My son is homeschooling and we are wondering what the process is to get approved since we don't have a school email. We have sent an official proof of homeschooling given by the local school board but that was not sufficient. Are there other alternatives?

Thanks

Erik.

**Comment by cmumatt** [comment]
@ErikApption It's not just you.  I am a traditional university student who has had an educational account for many years and can't get through the system either.  GitHub auto-closed all tickets about this today -- did that happen to you, too?

**Reply by ErikApption** [reply]
Mostly got automated emails and closed tickets. Not sure a real person ever read the request.
